services/ui-src/src/components/accordions/FaqAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+// components
+import { Accordion, Box, Text } from "@chakra-ui/react";
+import { AccordionItem } from "components";
+// utils
+import { AnyObject } from "types";
+
+export const FaqAccordion = ({ accordionItems, ...props }: Props) => {
+  return (
+    <Accordion allowToggle={true} allowMultiple={true} {...props}>
+      {accordionItems.map((item: AnyObject, index: number) => (
+        <AccordionItem key={index} label={item.question} sx={sx.item}>
+          <Box sx={sx.answerBox}>
+            <Text>{item.answer}</Text>
+          </Box>
+        </AccordionItem>
+      ))}
+    </Accordion>
+  );
+};
+
+interface Props {
+  accordionItems: AnyObject;
+  [key: string]: any;
```

Comment:
```suggestion
  [key: string]: any;
```

I don't think we need this line 😄

Reply:
nice catch! thank you karla! I will remove this line